tests/suites/test_suite_ecp.function

```diff
@@ -1390,7 +1390,46 @@ exit:
 }
 /* END_CASE */
 
-/* BEGIN_CASE depends_on:MBEDTLS_TEST_HOOKS */
+/* BEGIN_CASE depends_on:MBEDTLS_TEST_HOOKS:MBEDTLS_ECP_DP_CURVE448_ENABLED:MBEDTLS_ECP_WITH_MPI_UINT */
```

Comment:
I'm not sure why this has been resurrected - this is now handled by `ecp_mod_p_generic_raw`